tests/test_office_structure/test_delete_level.py

```diff
@@ -0,0 +1,23 @@
+from tests.base import BaseTestCase, CommonTestCases
+from fixtures.office_structure.delete_level_fixtures import (
+  delete_level_fixtures, delete_level_response,
+  delete_level_fixtures_non_existant_id
+)
+
+
+class TestDeleteQuestion(BaseTestCase):
```

Comment:
Hey @RachaelNantale , Kindly look at the naming of this class. I guess you meant `delete_level`